### PR TITLE
Updating workflows/data-fetching/sra-manifest-to-concatenated-fastqs from 0.2 to 0.3

### PR DESCRIPTION
**Pull request by gxydevbot**
Hello! This is an automated update of the following workflow: **workflows/data-fetching/sra-manifest-to-concatenated-fastqs**. I created this PR because I think one or more of the component tools are out of date, i.e. there is a newer version available on the ToolShed.

By comparing with the latest versions available on the ToolShed, it seems the following tools are outdated:
* `toolshed.g2.bx.psu.edu/repos/iuc/sra_tools/fasterq_dump/3.0.8+galaxy0` should be updated to `toolshed.g2.bx.psu.edu/repos/iuc/sra_tools/fasterq_dump/3.0.8+galaxy1`

The workflow release number has been updated from 0.2 to 0.3.

### CODE REVIEW

File: workflows/data-fetching/sra-manifest-to-concatenated-fastqs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [0.3] 2023-11-20
```
Comment (mvdbeek) [inline workflows/data-fetching/sra-manifest-to-concatenated-fastqs/CHANGELOG.md:3]:
```suggestion
## [0.2.1] 2023-11-20
```

File: workflows/data-fetching/sra-manifest-to-concatenated-fastqs/sra-manifest-to-concatenated-fastqs.ga [hunk 2]
```diff
@@ -19,7 +19,7 @@
         }
     ],
     "format-version": "0.1",
-    "release": "0.2",
+    "release": "0.3",
```
Comment (mvdbeek) [inline workflows/data-fetching/sra-manifest-to-concatenated-fastqs/sra-manifest-to-concatenated-fastqs.ga:22]:
```suggestion
    "release": "0.2.1",
```

**Review by mvdbeek — APPROVED** [review]

**Comment by lldelisle** [comment]
Should we change the tool which concatenate?

**Comment by mvdbeek** [comment]
If https://usegalaxy.org/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_cat/0.1.1 works for that task: 👍

**Comment by lldelisle** [comment]
Unfortunately the tp_cat tool does handle the list:list:pair... We need to keep this tool.

**Pull request merged**